internal_packages/keybase/lib/key-manager.cjsx

```diff
@@ -0,0 +1,35 @@
+{Utils, React} = require 'nylas-exports'
+PGPKeyStore = require './pgp-key-store'
+KeybaseUser = require './keybase-user'
+kb = require './keybase'
+_ = require 'underscore'
+
+module.exports =
+class KeyManager extends React.Component
+  @displayName: 'KeyManager'
+
+  @propTypes:
+    keys: React.PropTypes.array.isRequired
+
+  constructor: (props) ->
+    super(props)
+
+  render: ->
+    {keys} = @props
+
+    keys = _.map(keys, (key) =>
```

Comment:
This isn't super important, but we're trying to move toward using the built-in methods in ES6 `array.map`, `array.forEach` and `array.filter` —the underscore versions are a littler slower.